srcs/layouts/latn_qwerty_apl.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard name="QWERTY (APL)" script="latin" embedded_number_row="true">
+  <row height="0.75">
+    <key c="1" nw="⌶"    ne="¨"               se="!"/>
+    <key c="2" nw="¯"    ne="⍫"               se="@" sw="`"/>
+    <key c="3" nw="&lt;" ne="⍒"               se="#" sw="~"/>
+    <key c="4" nw="≤"    ne="⍋"               se="$"/>
+    <key c="5" nw="="    ne="⌽"               se="%"/>
+    <key c="6" nw="≥"    ne="⍉"               sw="^"/>
+    <key c="7" nw=">"    ne="⊖"               sw="&amp;" se="|"/>
+    <key c="8" nw="≠"    ne="⍟" se="∞"        sw="*"/>
+    <key c="9" nw="∨"    ne="⍱"               sw="("/>
+    <key c="0" nw="∧"    ne="⍲"               sw=")"/>
+  </row>
+  <row>
+    <key c="q" nw="?" ne="⍰"                  se="loc esc"/>
+    <key c="w" nw="⍵" ne="⍹"                  sw="'" se="&quot;"/>
+    <key c="e" nw="∊" ne="⍷"/>
+    <key c="r" nw="⍴"        sw="←" se="⇐"/>
+    <key c="t" nw="⍨"               se="→"/>
+    <key c="y" nw="↑" ne="⌹"        se="÷"    sw="-"/>
+    <key c="u" nw="↓" ne="⍮"        se="×"    sw="+"/>
+    <key c="i" nw="⍳" ne="⍸"                  sw="[" se="{"/>
+    <key c="o" nw="○" ne="⍥"                  sw="]" se="}"/>
+    <key c="p" nw="⍣" ne="⍬" sw="⍞" se="⍡"/>
+  </row>
+  <row>
+    <key c="a" nw="⍺" ne="⍶"                  sw="loc tab" shift="0.25"/>
+    <key c="s" nw="⌈" ne="ᑈ" se="«"/>
+    <key c="d" nw="⌊" ne="ᐵ" se="»"/>
+    <key c="f"  s="…"/>
+    <key c="g" nw="∇" ne="⍢"                  s="_"/>
+    <key c="h" nw="∆" ne="⍙"/>
+    <key c="j" nw="∘" ne="⍤" w="⊣" e="⊢"/>
+    <key c="k"        ne="⌸"/>
+    <key c="l" nw="⎕" ne="⌷" se="⍪"/>
+    <key c=";"  w="⌺"  n="⋄"                  s="," width="0.75"/>
+  </row>
+  <row>
+    <key c="shift" e="loc capslock"/>
```

Comment:
I think you meant to make this key wider:
```suggestion
    <key width="1.5" c="shift" e="loc capslock"/>
```

Reply:
I did intentionally do offsets as I did because otherwise I typo'd everything from being used to a staggered layout :)

yeah might make sense to follow your usual rules for a built-in layout.